packages/superset-ui-legacy-preset-chart-big-number/src/BigNumber/BigNumber.tsx

```diff
@@ -0,0 +1,309 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import shortid from 'shortid';
+import { OverlayTrigger, Tooltip } from 'react-bootstrap';
+import { t } from '@superset-ui/translation';
+import { getNumberFormatter } from '@superset-ui/number-format';
+import { XYChart, AreaSeries, CrossHair, LinearGradient } from '@data-ui/xy-chart';
+import { BRAND_COLOR } from '@superset-ui/color';
+import { computeMaxFontSize } from '@superset-ui/dimension';
+import NumberFormatter from '@superset-ui/number-format/lib/NumberFormatter';
+import { smartDateVerboseFormatter } from '@superset-ui/time-format';
+import TimeFormatter from '@superset-ui/time-format/lib/TimeFormatter';
+
+import './BigNumber.css';
+
+const defaultNumberFormatter = getNumberFormatter(undefined);
```

Comment:
Is it possible to omit `undefined`?

Reply:
Just wanted to be explicit. `getNumberFormatter` sounds like something that needs a parameter.